source/stylesheets/refills/unstyled/_switch.scss

```diff
@@ -0,0 +1,50 @@
+.switch {  
+  $switch-width: 52px;
+  $switch-padding: 2px;
+  $switch-height: 32px;
+  $knob-size: $switch-height - ($switch-padding * 2);
+  $knob-width: $knob-size;
+
+  cursor: pointer;
+  display: inline-block;
+  position: relative;
+
+  input[type="checkbox"] {
+    display: none;
+
+    + .checkbox {
+      @include size($switch-width $switch-height);
+      border: 1px solid #000;
```

Comment:
Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.